../../tests/invalid/MethodCall_Invalid_6.whiley:6: method invocation not permitted in function
    return {y: get(x), x: 1}
               ^^^^^^
